tests/test_weather_forecast.py

```diff
@@ -0,0 +1,79 @@
+import datetime
```

Comment:
Excellent job!! That's some advanced testing concepts you use here. Kudos!

Reply:
Wow, blushing...